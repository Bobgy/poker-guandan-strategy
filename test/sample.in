2
3
AD3D8H6HKCKS4DJC3H3C9H2C5H9SJD3CKH7CKCAH5H2C4C3SXRAC5S
6
KC0D4HKD6DJCKC2S8HADJD4D9C7SJH3SXBXR2DQH5H8S9D5S0S2D5C
