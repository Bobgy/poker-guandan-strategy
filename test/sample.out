cards=2H2HXRXR
mainRank=2
costMinPlays=1
cost=1
solutions=[
  '| XR XR WC WC |',
]

cards=AD3D8H6HKCKS4DJC3H3C9H2C5H9SJD3CKH7CKCAH5H2C4C3SXRAC5S
mainRank=3
costMinPlays=5
cost=4
solutions=[
  '| AC AD AH 2C 2C 3C 3C 3D 3S 4C 4D 5H 5S 7C 9S JC JD KC KC KH KS XR || 5H 6H 8H 9H WC |',
  '| AH 7C 9S JC JD KC KC KH KS XR || 3D 3S 4C 4D 5H 5S || AC AD 2C 2C 3C 3C || 5H 6H 8H 9H WC |',
  '| AC AD AH 3D 3S 5H 5S 7C 9S JC JD KC KC KH KS XR || 2C 2C 3C 3C 4C 4D || 5H 6H 8H 9H WC |',
  '| AC AD AH 2C 2C 3D 3S 7C 9S JC JD KC KC KH KS XR || 3C 3C 4C 4D 5H 5S || 5H 6H 8H 9H WC |',
]

cards=KC0D4HKD6DJCKC2S8HADJD4D9C7SJH3SXBXR2DQH5H8S9D5S0S2D5C
mainRank=6
costMinPlays=7
cost=7
solutions=[
  '| 2D 2S 4H 5H 5S JD JH KC KC KD XB XR || 8S 9D 0S JC QH || 6D 7S 8H 9C 0D || AD 2D 3S 4D 5C |',
  '| AD 2D 2D 2S 3S 4D 4H 5C 5H 5S JD JH KC KC KD XB XR || 8S 9D 0S JC QH || 6D 7S 8H 9C 0D |',
]

